usecases/auth/authorization/filter/filter.go

```diff
@@ -52,12 +54,24 @@ func (f *ResourceFilter[T]) Filter(
 	}
 
 	// For RBAC, filter based on per-item authorization
+	resources := []string{}
 	filtered := make([]T, 0, len(items))
 	for _, item := range items {
-		// TODO-RBAC: we need non silent authorizer with best effort enforcer
-		if err := f.authorizer.Authorize(principal, verb, resourceFn(item)); err == nil {
+		resources = append(resources, resourceFn(item))
+	}
+
+	allowedList, _ := f.authorizer.FilterAuthorizedResources(principal, verb, resources...)
```

Comment:
Should we at least log the error?

Reply:
yeah, didn't wanna expose logger to the filter, but it's inevitable now, let me add it